### DISCUSSION TITLE
Allow using `tests`

### DISCUSSION THREAD

**Discussion by ehsanmok** [DavidVujic/python-polylith#97] [Ideas]
Thank you for creating these plugins. I've become not only a fan but also an advocate of this. There's a bit of learning curve for sure but after that I feel I've a solution for better managing the structure of python mono-repos.

**Is your feature request related to a problem? Please describe.**

Please allow `tests` along with `test` where both are subjective but `tests` is more aligned when including unit-tests, functional and integ tests in the repo.

**Comment by DavidVujic** [comment]
Hi @ehsanmok! I'm happy to hear that you like Polylith and the tooling! 🤩 

Good feedback, maybe I should make that configurable. Meanwhile, you could disable the auto-creation of tests and add them manually where you would like them. Disabling can be done in the `workspace.toml` file:

```toml
[tool.polylith.test]
enabled = false
```

**Comment by DavidVujic** [comment]
I'll move this to the discussions - ideas section.
